x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/enrollment_instructions/manual/index.tsx

```diff
@@ -20,37 +20,79 @@ export const ManualInstructions: React.FunctionComponent<Props> = ({
   apiKey,
   kibanaCASha256,
 }) => {
-  const command = `
-./elastic-agent enroll ${kibanaUrl} ${apiKey.api_key}${
+  const enrollArgs = `${kibanaUrl} ${apiKey.api_key}${
     kibanaCASha256 ? ` --ca_sha256=${kibanaCASha256}` : ''
-  }
+  }`;
+  const macOsLinuxTarCommand = `
+./elastic-agent enroll ${enrollArgs}
 ./elastic-agent run`;
+
+  const linuxDebRpmCommand = `
+./elastic-agent enroll ${enrollArgs}
+systemctl enable elastic-agent
+systemctl start elastic-agent`;
+
+  const windowsCommand = `
+./elastic-agent enroll ${enrollArgs}
+./install-service-elastic-agent.ps1`;
```

Comment:
```suggestion
  const macOsLinuxTarCommand = `./elastic-agent enroll ${enrollArgs}
./elastic-agent run`;

  const linuxDebRpmCommand = `./elastic-agent enroll ${enrollArgs}
systemctl enable elastic-agent
systemctl start elastic-agent`;

  const windowsCommand = `./elastic-agent enroll ${enrollArgs}
./install-service-elastic-agent.ps1`;
```

Reply:
I can't comment to formal style guides, but it feels more readable to me here as committed.

Reply:
this suggestion doesn't affect how the commands are displayed in the UI. however this suggestion is probably no longer relevant due to https://github.com/elastic/kibana/pull/73491#discussion_r461818718